docs/GRAPHITE_SETUP.md

```
@@ -0,0 +1,221 @@
+
+# Move up one branch
+gt up
+
+# Move down one branch
+gt down
+```
+
+## Recommended Workflow
+
+1. **Start from main**:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)